Storage/SQL/SQLiteBookmarksModel.swift

```diff
@@ -612,18 +626,22 @@ class BookmarkFactory {
         return separator
     }
 
-    fileprivate class func itemFactory(_ row: SDRow) -> BookmarkItem {
+    fileprivate class func itemRowFactory(_ row: SDRow, forceEditable: Bool = false) -> BookmarkItem {
         let id = row["id"] as! Int
         let guid = row["guid"] as! String
         let url = row["bmkUri"] as! String
         let title = row["title"] as? String ?? url
-        let isEditable = row.getBoolean("isEditable")           // Defaults to false.
+        let isEditable = forceEditable || row.getBoolean("isEditable")           // Defaults to false.
         let bookmark = BookmarkItem(guid: guid, title: title, url: url, isEditable: isEditable)
         bookmark.id = id
         BookmarkFactory.addIcon(bookmark, row: row)
         return bookmark
     }
 
+    fileprivate class func itemFactory(_ row: SDRow) -> BookmarkItem {
```

Comment:
This has to exist because the type signature for a function with optional arguments omitted is not the same as a function without those arguments.